packages/react/src/stories/ic-text-field.stories.mdx

```diff
@@ -165,6 +165,32 @@ export const ControlledExample = () => {
   </Story>
 </Canvas>
 
+### Max length message
+
+<Canvas>
+  <Story name="Max length message">
+    <IcTextField
+      maxLength="25"
+      value="Arabica"
+      label="What is your favourite coffee?"
+      required
+      placeholder="Placeholder"
+      helperText="Such as Arabica, Robusta or Liberica"
+      maxLengthMessage="You have reached the maximum length"
```

Comment:
Small quibble but I think the validation messages should read 'You have exceeded the maximum length'
This applies for the web-components stories also.